app/app.go

```diff
@@ -376,19 +377,19 @@ func New(
 		genutil.NewAppModule(app.AccountKeeper, app.StakingKeeper, app, encodingConfig.TxConfig),
 		auth.NewAppModule(encodingConfig.Codec, app.AccountKeeper, authsims.RandomGenesisAccounts, app.GetSubspace(authtypes.ModuleName)),
 		vesting.NewAppModule(app.AccountKeeper, app.BankKeeper),
-		bank.NewAppModule(encodingConfig.Codec, app.BankKeeper, app.AccountKeeper, app.GetSubspace(banktypes.ModuleName)),
+		bankModule{bank.NewAppModule(encodingConfig.Codec, app.BankKeeper, app.AccountKeeper, app.GetSubspace(banktypes.ModuleName))},
```

Comment:
this feels super hacky, is there a better way to support the desired logic? (overriding `DefaultGenesis` behaviour)

Reply:
isn't there a cleaner way we can do overrides? 🤔

Reply:
no that's the way to do it without forking.